java-metric-example/src/main/java/com/vmware/OtelService.java

```diff
@@ -0,0 +1,115 @@
+package com.vmware;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.metrics.LongHistogram;
+import io.opentelemetry.exporter.otlp.metrics.OtlpGrpcMetricExporter;
+import io.opentelemetry.sdk.metrics.InstrumentSelector;
+import io.opentelemetry.sdk.metrics.InstrumentType;
+import io.opentelemetry.sdk.metrics.SdkMeterProvider;
+import io.opentelemetry.sdk.metrics.SdkMeterProviderBuilder;
+import io.opentelemetry.sdk.metrics.View;
+import io.opentelemetry.sdk.metrics.export.AggregationTemporalitySelector;
+import io.opentelemetry.sdk.metrics.export.MetricReader;
+import io.opentelemetry.sdk.metrics.export.PeriodicMetricReader;
+import io.opentelemetry.sdk.metrics.internal.view.ExplicitBucketHistogramAggregation;
+import io.opentelemetry.sdk.metrics.internal.view.ExponentialHistogramAggregation;
```

Comment:
I'm concerned that we're importing internal packages – it makes me think they're not meant for users of the SDK, no? I see this [PR](https://github.com/open-telemetry/opentelemetry-java/pull/4472) was opened two days ago, allowing to configure a `MetricReader` and `MetricExporter` to use exponential histograms by default. Perhaps that's the non-internal API to use? We'd need to wait for the PR to be merged of course, before we can consume that API.

Reply:
Lets hold on to this PR, we will wait for Jack's PR to be merged.